app/models/category.rb

```diff
@@ -2,4 +2,12 @@ class Category < ActiveRecord::Base
   has_many :category_items
   has_many :items, through: :category_items
   validates :name, presence: true, uniqueness: true
+
+  def self.search(search)
+    if search
+      where('name LIKE ?', "%#{search}%")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.